edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/SampleDataLoader.java

```diff
@@ -477,8 +487,8 @@ protected void loadDiscussionSection(String secEidPrefix, String asEid, String c
 		if(log.isDebugEnabled()) log.debug("Created section " + secEid);
 	}
 
-	protected int incrementStudentCount() {
-		studentMemberCount += 30;
+	protected int incrementStudentCount(boolean mega) {
```

Comment:
I would make this more flexible and have it take an integer instead of a boolean, and use that value to increment the count.

Reply:
Sure.